brave-grpc/README.md

```diff
@@ -0,0 +1,20 @@
+# brave-jersey #
+
+Contributed by [Eric Hauser](https://github.com/ewhauser).
```

Comment:
I think this is copy/paste, but usually we don't add this (even if jersey did). we still love you, though!